api/v1alpha4/awscluster_webhook.go

```diff
@@ -104,6 +105,13 @@ func (r *AWSCluster) ValidateUpdate(old runtime.Object) error {
 		)
 	}
 
+	if annotations.IsExternallyManaged(oldC) && !annotations.IsExternallyManaged(r) {
```

Comment:
Do we want to check the other way around as well?

Reply:
My understanding is that the user can add this annotation and disable infrastructure management, am I wrong?

Reply:
Going from managed to externally managed should should be allowed. The other way around is not allowed.